README.md

```diff
@@ -13,19 +12,17 @@ See [docs/README.md](./docs/README.md).
 
 Channel        | Latest Build
 ---------------|:---------------
-master         | ![badge][master-badge]
+main           | ![badge][main-badge]
 release/2.1    | ![badge][rel-2.1-badge]
-release/2.0    | ![badge][rel-2.0-badge]
 
-[master-badge]: https://aspnetcore.blob.core.windows.net/buildtools/korebuild/channels/master/badge.svg
+[main-badge]: https://aspnetcore.blob.core.windows.net/buildtools/korebuild/channels/main/badge.svg
```

Comment:
This channel won't be populated until #1019 is merged. Will check after that goes in to be sure the official build updates the blob store.